src/main/scala/ch/epfl/bluebrain/nexus/kg/async/ProjectViewCoordinatorActor.scala

```diff
@@ -110,15 +110,15 @@ private abstract class ProjectViewCoordinatorActor(viewCache: ViewCache[Task])(i
         "organization" -> project.organizationLabel
       )
     val count = AtomicLong(0L)
-    SequentialTagIndexer.start(
-      IndexerConfig
+    TagProjection.start(
+      ProjectionConfig
         .builder[Task]
         .name(s"project-event-count-${project.uuid}")
         .tag(s"project=${project.uuid}")
         .plugin(config.persistence.queryJournalPlugin)
-        .retry[ElasticSearchServerOrUnexpectedFailure](indexing.retry.retryStrategy)
+        .retry[ElasticSearchServerOrUnexpectedFailure](indexing.retry.retryStrategy)(elasticErrorMonadError)
         .batch(indexing.batch, indexing.batchTimeout)
-        .restart(false)
+        .restart(value = false)
```

Comment:
even though IntellIj does not like to have booleans without specifying the parameter, in this case it doesn't make sense, since it is a builder. The information is already on the method (restart(true) or restart(false)).

Reply:
agreed